Gradle build error

I'm seeing the following error while trying to build the current main branch:
```
FAILURE: Build failed with an exception.

* Where:
Build file '***/paccor/build.gradle' line: 118

* What went wrong:
A problem occurred evaluating root project 'paccor'.
> Could not get unknown property 'all' for object of type com.netflix.gradle.plugins.packaging.ProjectPackagingExtension.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
```
Modifying this line:
https://github.com/nsacyber/paccor/blob/e05b67688177c809bb3d598b57a05999f9152fbc/build.gradle#L118
to
```
 arch='all'
```
appears to fix the issue. I don't really know JAVA so it's entirely possible this is not a proper fix.

--- a/build.gradle
+++ b/build.gradle
@@ -115,9 +115,8 @@ applicationDistribution.from("./") {
 ospackage {
     packageName='paccor'
     os=LINUX
-    arch=all
     version='1.1.4'
-    release='6'
+    release='7'
 
     into '/opt/paccor'
     user 'root'

